java-kms/README.md

```diff
@@ -11,59 +11,28 @@ Java idiomatic client for [Cloud Key Management Service][product-docs].
 
 ## Quickstart
 
-If you are using Maven with [BOM][libraries-bom], add this to your pom.xml file:
 
-```xml
```

Comment:
@suztomo Do you know why the README is changing so much?

Reply:
It's based on samples and we deleted the samples. More in https://github.com/googleapis/google-cloud-java/issues/8537#issuecomment-1270050251

Reply:
Ah right!